changelog/updates/2022-01-11-profiles-update.md

```diff
@@ -0,0 +1,2 @@
+- pciutils ([3.7.0](https://github.com/pciutils/pciutils/commit/864aecdea9c7db626856d8d452f6c784316a878c))
+- usbutils ([014](https://github.com/gregkh/usbutils/commit/57fb18e59cce31a50a1ca62d1e192512c905ba00))
```

Comment:
We could add an entry too for `lshw` which is a user-facing program ?

Reply:
I thought about it, but the ebuild update actually did not bring any new version of lshw. I actually updated it very recently.

Reply:
oh ok - nice then :)